### PR TITLE
TZ011 - Define domain entities/dto in OrderManagementService

### PR DESCRIPTION
**Pull request by angheladriana**
Created entities and dto in OrderManagement

### CODE REVIEW

File: tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/OrderDto.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class OrderDto {
+    private Long id;
```
Comment (teddykitten) [inline tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/OrderDto.java:14]: fara id in dto

**Review by teddykitten — APPROVED** [review]

File: tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/CartDto.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class CartDto {
+    private String uuid;
+    private String vendor_uuid;
+    private String user_uuid;
+    private List<CartItemDto> carts;
```
Comment (cristianiova) [inline tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/CartDto.java:14]: Field name should be cartItems.

File: tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/OrderDto.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class OrderDto {
+    private String uuid;
+    private LocalDateTime date;
+    private BigDecimal total;
+    private List<OrderItemDto> orders;
```
Comment (cristianiova) [inline tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/model/OrderDto.java:17]: Field name should be orderItems.

File: tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/persistence/entity/Cart.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    @Column(name = "vendor_uuid", nullable = false, unique = true)
+    private String vendor_uuid;
+
+    @Column(name = "user_uuid", nullable = false, unique = true)
+    private String user_uuid;
+
+    @Column(name = "cart_item_id", nullable = false)
+    @OneToMany(fetch = FetchType.EAGER)
+    @JoinColumn(name = "id")
+    private List<CartItem> carts;
```
Comment (cristianiova) [inline tazzan/orderManagement/src/main/java/com/nagarro/tazzan/ordermanagement/persistence/entity/Cart.java:31]: Field name should be cartItems.

**Comment by cristianiova** [comment]
Please do not use _ in fields name.
e.g. product_uuid -> productUuid

**Review by teddykitten — APPROVED** [review]

**Pull request merged**